docs/src/snap/explanation/ingress.md

```diff
@@ -13,59 +29,48 @@ they are exposed through services.
 Services offer a stable network interface by providing discoverable names and
 load balancing functionality for managing a set of pods.
 For further details on Kubernetes Services,
-refer to the upstream documentation: [Kubernetes Service][Service].
+refer to the [upstream Kubernetes Service documentation][Service].
 
 ## Ingress
 
-[Ingress][Ingress K8s] is a Kubernetes Resource that manages
+[Ingress][Ingress K8s] is a Kubernetes resource that manages
 external access by handling both HTTP and HTTPS traffic to services within your cluster.
 Traffic routed through the Ingress is directed to a service,
 which in turn forwards it to the relevant pod
 running the desired application within a container.
 
-The Ingress Resource lets you define rules on how traffic should get handled.
-Learn more about these rules and how to implement them here:
-[Ingress Rules][Ingress Rules].
+The Ingress resource lets you define rules on how traffic should get handled.
+Refer to the [Kubernetes documentation on Ingress rules][Ingress Rules]
+for up to date information on the available rules and their implementation.
+
 
-While the Ingress Resource manages the routing rules for the incoming traffic,
+While the Ingress resource manages the routing rules for the incoming traffic,
 the [Ingress Controller][Ingress Controller] is responsible for implementing
 those rules by configuring the underlying networking infrastructure of the cluster.
 Ingress does not work without an Ingress Controller.
 
 The Ingress Controller also serves as a layer 7 (HTTP/HTTPS) load balancer
 that routes traffic from outside of your cluster to services inside of your cluster.
-Please do not confuse this with the Kubernetes Service LoadBalancer type.
+Please do not confuse this with the Kubernetes Service LoadBalancer type
+which operates at layer 4 and routes traffic directly to individual pods.
 
 ```{kroki} ../../assets/ingress.puml
 ```
 
 With Canonical Kubernetes, enabling default Ingress is easy:
-See [default Ingress][Ingress].
-At the moment of writing, enabling default Ingress will create a
-[Cilium Ingress Controller][Cilium Ingress Controller] for you.
+See the [default Ingress guide][Ingress].
+Enabling default Ingress will create an
+[Ingress Controller][Cilium Ingress Controller] for you.
+The underlying mechanism provided by default will always operate in the same way,
+and will be Cilium for the foreseeable future.
+However, we do reserve the right to change it in the future if necessary.
```

Comment:
```suggestion
The underlying mechanism provided by default is currently Cilium. However, it should always be operated through the provided CLI rather than directly. This way, we can provide the best experience for future cluster maintenance and upgrades.
```

Reply:
Much better put, thanks! @perk